libs/platform/src/lib/platform.module.ts

```diff
@@ -1,12 +1,10 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { FundamentalNgxCoreModule } from '@fundamental-ngx/core';
+import { PlatformButtonModule } from './components/button/button.module';
 
 @NgModule({
-    imports: [
-        CommonModule,
-        FundamentalNgxCoreModule
-    ]
+    imports: [CommonModule, FundamentalNgxCoreModule],
```

Comment:
Dont change the existing formating to be inlined. 

Check your formating rules, they should not inline all like this it should be like before where arrays like imports are on the new lines, 

the same for class. declaratin

`{}` vs 

```{
}
``

Reply:
ng lint error is occurring if I place in a new line.

Reply:
it should not, what exactly is the linting error? we havent changed linting rules and they passed before..

Reply:
> it should not, what exactly is the linting error? we havent changed linting rules and they passed before..

Done, aligned in new line.